### ISSUE TITLE
Fails to parse "S28803240P4K27GA00003PB000100DXD   0000499600000N4AQ00003"

### ISSUE THREAD

**Issue by bigfatwhale** [bigfatwhale/bats_pitch_parser#11]
```
Chu-Mings-iMac:sandbox unclechu$ ipython
Python 2.7.12 |Anaconda 2.3.0 (x86_64)| (default, Jul  2 2016, 17:43:17) 
Type "copyright", "credits" or "license" for more information.

IPython 5.3.0 -- An enhanced Interactive Python.
?         -> Introduction and overview of IPython's features.
%quickref -> Quick reference.
help      -> Python's own help system.
object?   -> Details about 'object', use 'object??' for extra details.

In [1]: import bats_api

In [2]: parser = bats_api.BATSPitchMsgParser()

In [3]: x=parser.parse_msg("S28803240P4K27GA00003PB000100DXD   0000499600000N4AQ00003")

In [4]: x

In [5]: x is None
Out[5]: True

In [6]: 
```

**Comment by bigfatwhale** [comment]
Simple smoke test code,
```python

import bats_api

parser = bats_api.BATSPitchMsgParser()

with open( 'pitch_example_data' ) as f:
	for line in f:
		line = line.strip()
		msg = parser.parse_msg(line)
		if msg is None:
			raise Exception('parse failure for line : %s' % line)
		print msg
```
 dies on parsing this specific msg, 

```
BATSOrderCancelMsg(timestamp=28803234, msgtype=X, orderId=599834127447466117, shares=100)
BATSOrderCancelMsg(timestamp=28803238, msgtype=X, orderId=599834127447466108, shares=47)
BATSAddOrderMsg(timestamp=28803238, msgtype=A, orderId=599834127447466118, side=B, shares=200, symbol=DIA   , price=1298100, display=Y, partId=)
Traceback (most recent call last):
  File "test_parser.py", line 11, in <module>
    raise Exception('parse failure for line : %s' % line)
Exception: parse failure for line : S28803240P4K27GA00003PB000100DXD   0000499600000N4AQ00003
```

**Comment by bigfatwhale** [comment]
Fixed in [8ce6e1d](https://github.com/bigfatwhale/bats_pitch_parser/commit/8ce6e1d9565aa28206f4d11470efc28544fe7039)
